docs/topics/installing-vs-code-plugin.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// * docs/vs-code-plugin-guide/master.adoc
+
+
+[id="installing-vs-code-plugin_{context}"]
+= Installing the {VscodePluginName} extension
+
+You can install the {VscodePluginName} ({VscodePluginShortName}) extension on Linux, Windows, or macOS.
+
+As of June 2021, VS Code has been tested on the following platforms:
```

Comment:
Is this for VS Code or the extension? If it is for VS Code, we should not be documenting it.